config/samples/backends/README.md

```diff
@@ -20,29 +24,74 @@ Currently available samples are:
 - NFS
 - LVM using iSCSI
 - LVM using NVMe-TCP
+- HPE 3PAR iSCSI
+- HPE 3PAR FC
+- NetApp ONTAP iSCSI
+- NetApp ONTAP FC
+- NetApp ONTAP NFS
+- Pure Storage iSCSI
+- Pure Storage FC
+- Pure Storage NVMe-RoCE
 
-## Ceph example
+These examples are designed to be applied one at a time, you cannot get 2
```

Comment:
@ASBishop here is where I've added the comment of this being single backend samples.

Reply:
I'd like to see something more like this (for your consideration):

NOTE: These examples are designed to be applied one at time. If you attempt to add a second backend by applying a second example, it will result in the first backend being removed. It is certainly possible to define a deployment with multiple backends, but that requires a single CR that defines all backends.

Reply:
Done